src/main/java/io/github/externschool/planner/repository/RoleRepository.java

```diff
@@ -1,12 +1,11 @@
 package io.github.externschool.planner.repository;
 
-import io.github.externschool.planner.entity.Authority;
+import io.github.externschool.planner.entity.Role;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface AuthorityRepository extends CrudRepository<Authority, String> {
-
-    Authority findByName(String name);
+public interface RoleRepository extends CrudRepository<Role, String> {
```

Comment:
Why did you choose Crud instead Jpa.
I think Jpa is better and we will delete some code from our projects.

Jpa supports out of the box 
pagination and sorting and some other methods

For example for findAll Jpa returns list instead iterable